### PR TITLE
envoy.dependency.check: Update checker and pass through args

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

**Comment by phlax** [comment]
cc @kfaseela

File: envoy.dependency.check/tests/test_integration_cves.py [hunk 1]
```diff
@@ -64,8 +65,12 @@ def test_integration_cves(
          dict(new_callable=PropertyMock)),
         ("DependencyChecker.log",
          dict(new_callable=PropertyMock)),
+        ("DependencyChecker.loop",
+         dict(new_callable=PropertyMock)),
         ("DependencyChecker.session",
          dict(new_callable=PropertyMock)),
+        "DependencyChecker.install_reactor",
+        "DependencyChecker.setup_logging",
```
Comment (phlax) [inline envoy.dependency.check/tests/test_integration_cves.py:73]: this bit is slightly unrelated to the pr - other than the changes to aio.run.checker broke this (on current main)

**Review by kfaseela — APPROVED** [review]
Change looks good overall, may be you just need to fix the type check :)

**Comment by phlax** [comment]
> may be you just need to fix the type check :)

yep, on it, thanks for review

**Pull request merged**